IntelliJ Project/src/main/java/com/globant/musicstore/entity/Artist.java

```diff
@@ -35,11 +34,8 @@
     @Column(name = "is_active", nullable = false)
     private boolean isActive;
 
-    @ManyToOne(cascade = CascadeType.ALL, optional = true, fetch = FetchType.EAGER)
+    @ManyToOne(fetch = FetchType.EAGER)
     @JoinColumn(name = "house_record_id", referencedColumnName = "house_record_id")
     @JsonIgnoreProperties("artists")
     private HouseRecord houseRecord;
-
-    @OneToMany(mappedBy = "artist")
```

Comment:
We need to put the bidirectionality, OneToMany and ManyToOne in the Artist Entity and AlbumEntity, respectively